docs-source/content/userguide/managing-domains/model-in-image/model-files.md

```diff
@@ -84,7 +84,7 @@ Some notes about the sample model file:
 
 #### Model file naming and loading order
 
-Refer to this section if you need to control the order in which your model files are loaded.
+Refer to this section if you need to control the order in which your model files are loaded.  The order is important when two or model files refer to the same configuration, as the the last model that's loaded has the highest precedence.
```

Comment:
as the the last model ->  because the the last model

Reply:
@tbarnes-us Sorry. The edit is, as the the last model -> because the the last model